src/datasources/test-plans/types.ts

```diff
@@ -88,3 +193,38 @@ export const OrderByOptions = {
     ID: 'ID',
     UPDATED_AT: 'UPDATED_AT'
 };
```

Comment:
Should mark exported const objects `as cont`

```suggestion
} as const;
```

Reply:
Applied suggestion.